README.md

```diff
@@ -117,7 +117,6 @@ To find available packages that you can add to your configuration:
 shelffiles/
 ├── config/           # Configuration files
 │   └── nix/          # Nix-related configuration
-│       └── packages.nix  # Package definitions
```

Comment:
package.nix is moved to under shelffiles, so you should add new line to this chart